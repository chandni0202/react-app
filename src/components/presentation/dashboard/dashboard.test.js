import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from './dashboard.jsx';

//snapshot test case to check page renders correctly
// run npm test -- -u if do any changes in page view
it('renders correctly', () => {
  const tree = renderer
    .create(<Dashboard/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});