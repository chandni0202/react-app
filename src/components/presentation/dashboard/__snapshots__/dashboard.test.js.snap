// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <table>
    <tbody>
      <tr>
        <th>
          Name
          <span
            className="name"
            onClick={[Function]}
          />
        </th>
        <th>
          UserName
        </th>
        <th>
          Address
        </th>
        <th>
          Phone
        </th>
        <th>
          Email
          <span
            className="email"
            onClick={[Function]}
          />
        </th>
        <th>
          Website
        </th>
        <th>
          Company
        </th>
        <th>
          Action
        </th>
      </tr>
    </tbody>
  </table>,
  <div
    className="pagination_container"
  >
    <select
      className="select-item"
      defaultValue="4"
      id="select-item"
      onChange={[Function]}
    >
      <option
        value="2"
      >
        2 Per Page
      </option>
      <option
        value="4"
      >
        4 Per Page
      </option>
      <option
        value="6"
      >
        6 Per Page
      </option>
      <option
        value="8"
      >
        8 Per Page
      </option>
      <option
        value="10"
      >
        10 Per Page
      </option>
      <option
        value="20"
      >
        20 Per Page
      </option>
      <option
        value="50"
      >
        50 Per Page
      </option>
    </select>
    <div
      style={
        Object {
          "display": "flex",
          "margin": "-2em",
        }
      }
    >
      <div
        className="pagination"
      >
        <button
          disabled={true}
          onClick={[Function]}
        >
          previous
        </button>
        <button
          id="next"
          onClick={[Function]}
        >
          next
        </button>
      </div>
    </div>
  </div>,
]
`;
